src/layouts/Settings.tsx

```diff
@@ -110,6 +111,10 @@ const Settings = () => {
                 link: INTEGRATIONS_ROUTE,
                 hidden: !hasPermissions(['organizationIntegrationsView']),
               },
+              {
+                title: translate('text_664c732c264d7eed1c74fd96'),
+                link: AUTHENTICATION_ROUTE,
```

Comment:
Since you merged rbac logic, should I add permission check here too @ansmonjol ?

Reply:
Good point, you probably need to check with product. I'm not sure such permissions exists today. Feel it's supposed to exist tho.